tap_mailchimp/client.py

```diff
@@ -22,6 +23,14 @@ def __init__(self, config):
         self.__base_url = None
         self.page_size = int(config.get('page_size', '1000'))
 
+        # performs date-window calculation for fetching campaigns
+        try:
+            date_window_duration = int(config.get('date_window', 0))
+            self.adjusted_start_date = False if date_window_duration == 0 else \
+                (singer.utils.now().date() - singer.utils.datetime.timedelta(days = date_window_duration))
+        except ValueError:
+            LOGGER.info("Invalid Value: %s, for date windowing", config.get('date_window', 0))
+            self.adjusted_start_date = False
```

Comment:
I feel like we should error here, since something strange has made it into our config. At the very least, we should be logging that we are reverting to not using date windowing if thats what we decide to do.

Reply:
this misconfiguration can be handled, so erroring on this does not feel required, hence added a critical log statement.